source/blog/2022-02-10-announcing-hanami-200alpha6.html.markdown

```diff
@@ -0,0 +1,166 @@
+---
+title: Announcing Hanami v2.0.0.alpha6
+date: 2022-02-10 11:04:32 UTC
+tags: announcements
+author: Luca Guidi
+image: true
+excerpt: >
+  Hanami Container Providers, Partial Slice Import/Export. Ruby 3.0+ only.
```

Comment:
I think we could probably just call them "Providers" rather than "Container providers"? It really depends on how forward we want to be with the "container" terminology when we describe the framework to our users.

Reply:
Certainly "Container providers" is _not_ the correct term for them if we were talking about them at a dry-system level. They're just "providers".